packages/db/test/pinning.spec.js

```diff
@@ -114,9 +116,10 @@ describe('Pin Request', () => {
       assert.strictEqual(savedPinRequest._id, aPinRequestOutput._id)
     })
 
-    it('returns the right object', async () => {
+    it.only('returns the right object', async () => {
```

Comment:
```suggestion
    it('returns the right object', async () => {
```

I would also welcome a better test name here...

Reply:
Thanks just fixed that, will rename